src/nl/help/guides/getting-started.md

```diff
@@ -8,89 +8,98 @@ lang: nl-NL
 ## Installatie
 **Paperback** is momenteel _niet_ op **TestFlight** en _vereist_ het gebruik van **AltStore** om op je iPhone of iPad te installeren. In de toekomst zal TestFlight worden gebruikt wanneer de hoofdontwikkelaar een [Apple Developer Account](https://developer.apple.com/programs/) heeft.
 
-::: tip Video tutorial
-Als je liever een video-tutorial gebruikt, heeft Artesians er een mooie gemaakt. Je vindt deze [hier](https://www.youtube.com/watch?v=n1KRwsxNiWY).
+::: tip Video-tutorial
+Als je liever een video-tutorial gebruikt, heeft Artesians er een gemaakt. Je vindt deze [hier](https://www.youtube.com/watch?v=n1KRwsxNiWY).
```

Comment:
You should add macOS tutorial: https://www.youtube.com/watch?v=CjPjsF4yJ0M

Reply:
Added a link for macOS, thanks Framboise